Cargo.toml

```diff
@@ -32,8 +32,10 @@ zeroize = { version = "1", optional = true,  default-features = false }
 bincode = "1"
 criterion = { version = "0.5", features = ["html_reports"] }
 hex-literal = "0.4"
-num-bigint = { package = "num-bigint-dig", version = "0.8" }
+num-bigint-dig = "0.8"
+num-bigint = "0.4"
```

Comment:
Is there a particular reason we need both?

Reply:
That's exactly my question. Why was the `-dig` one used originally? I assumed it was because it had a better performance, so it was used as a reference in Boxed benchmarks.

Reply:
If we can migrate fully to `num-bigint` we should. It was just simpler to start with `num-bigint-dig` to implement the gaps for `rsa` since it's what `rsa` already used

Reply:
So some preliminary tests: for `U1024+Limb`, const-time GCD takes 11us, and `gcd_limb_vartime()` takes 264ns. `gcd_vartime()` is quite slow though, because `rem_vartime()` is actually not much faster than `rem()`.